### ISSUE TITLE
Crash on toolbar open

### ISSUE THREAD

**Issue by vlad20012** [duxtinto/github-cr-intellij-plugin#23]
I've just installed the plugin, set up github token, opened toolar and got the crash:
```
project.getComponent(TaskManager::class.java) must not be null
java.lang.IllegalStateException: project.getComponent(TaskManager::class.java) must not be null
	at com.duxtinto.intellij.plugin.github.codereviews.di.impl.dagger.modules.idea.IdeaTaskModule.provideTaskManager(IdeaTaskModule.kt:14)
	at com.duxtinto.intellij.plugin.github.codereviews.di.impl.dagger.modules.idea.IdeaTaskModule_ProvideTaskManagerFactory.get(IdeaTaskModule_ProvideTaskManagerFactory.java:28)
	at com.duxtinto.intellij.plugin.github.codereviews.di.impl.dagger.modules.idea.IdeaTaskModule_ProvideTaskManagerFactory.get(IdeaTaskModule_ProvideTaskManagerFactory.java:10)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at com.duxtinto.intellij.plugin.github.codereviews.ide.acl.services.tasks.IdeaTaskActivator_Factory.get(IdeaTaskActivator_Factory.java:27)
	at com.duxtinto.intellij.plugin.github.codereviews.ide.acl.services.tasks.IdeaTaskActivator_Factory.get(IdeaTaskActivator_Factory.java:9)
	at com.duxtinto.intellij.plugin.github.codereviews.domain.issues.SwitchToIssueInteractor_Factory.get(SwitchToIssueInteractor_Factory.java:21)
	at com.duxtinto.intellij.plugin.github.codereviews.domain.issues.SwitchToIssueInteractor_Factory.get(SwitchToIssueInteractor_Factory.java:7)
	at com.duxtinto.intellij.plugin.github.codereviews.presentation.pullrequestlist.events.PullRequestListMouseListener_Factory.get(PullRequestListMouseListener_Factory.java:23)
	at com.duxtinto.intellij.plugin.github.codereviews.presentation.pullrequestlist.events.PullRequestListMouseListener_Factory.get(PullRequestListMouseListener_Factory.java:8)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at com.duxtinto.intellij.plugin.github.codereviews.presentation.pullrequestlist.PullRequestListView_Factory.get(PullRequestListView_Factory.java:33)
	at com.duxtinto.intellij.plugin.github.codereviews.presentation.pullrequestlist.PullRequestListView_Factory.get(PullRequestListView_Factory.java:10)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at com.duxtinto.intellij.plugin.github.codereviews.presentation.reviewee.RevieweeView_Factory.get(RevieweeView_Factory.java:33)
	at com.duxtinto.intellij.plugin.github.codereviews.presentation.reviewee.RevieweeView_Factory.get(RevieweeView_Factory.java:10)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at com.duxtinto.intellij.plugin.github.codereviews.ide.acl.presentation.toolwindow.ToolWindowContentPresenter_Factory.get(ToolWindowContentPresenter_Factory.java:61)
	at com.duxtinto.intellij.plugin.github.codereviews.ide.acl.presentation.toolwindow.ToolWindowContentPresenter_Factory.get(ToolWindowContentPresenter_Factory.java:13)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
	at com.duxtinto.intellij.plugin.github.codereviews.di.impl.dagger.DaggerApplicationContainer$ProjectContainerImpl.injectToolWindowFactory(DaggerApplicationContainer.java:585)
	at com.duxtinto.intellij.plugin.github.codereviews.di.impl.dagger.DaggerApplicationContainer$ProjectContainerImpl.inject(DaggerApplicationContainer.java:572)
	at com.duxtinto.intellij.plugin.github.codereviews.ide.acl.presentation.toolwindow.ToolWindowFactory.createToolWindowContent(ToolWindowFactory.kt:20)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.ensureContentInitialized(ToolWindowImpl.java:573)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.getContentManager(ToolWindowImpl.java:407)
	at com.intellij.openapi.wm.impl.commands.RequestFocusInToolWindowCmd.processRequestFocus(RequestFocusInToolWindowCmd.java:73)
	at com.intellij.openapi.wm.impl.commands.RequestFocusInToolWindowCmd.lambda$run$0(RequestFocusInToolWindowCmd.java:61)
	at com.intellij.openapi.util.ExecutionCallback.doWhenExecuted(ExecutionCallback.java:100)
	at com.intellij.openapi.util.ActionCallback.doWhenDone(ActionCallback.java:111)
	at com.intellij.openapi.wm.impl.commands.RequestFocusInToolWindowCmd.run(RequestFocusInToolWindowCmd.java:61)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.ToolWindowsPane$AddDockedComponentCmd.run(ToolWindowsPane.java:768)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:93)
	at com.intellij.openapi.wm.impl.CommandProcessor.run(CommandProcessor.java:74)
	at com.intellij.openapi.wm.impl.commands.FinalizableCommand.finish(FinalizableCommand.java:36)
	at com.intellij.openapi.wm.impl.commands.ApplyWindowInfoCmd.run(ApplyWindowInfoCmd.java:50)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:424)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:407)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:822)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:650)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:366)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by vlad20012** [comment]
Oh, ok, I don't have TaskManager plugin. I've removed it manually some time ago =D

**Comment by duxtinto** [comment]
hi @vlad20012 ! 
Thanks for letting me know this error... as the TaskManager plugin is not essential for this code review plugin main business, I'm going to make it work even if the TaskManager plugin has been removed.

**Comment by duxtinto** [comment]
Hi @vlad20012 !

this crash has been fixed by the v1.0.2 release.
Until it's available in the Jetbrains' repository, you can download it from here: https://github.com/duxtinto/github-cr-intellij-plugin/releases/tag/v1.0.2

Let me know if you have any other issue. ok?
